presto-hive/src/main/java/com/facebook/presto/hive/PrismTransactionHandle.java

```diff
@@ -0,0 +1,17 @@
+package com.facebook.presto.hive;
+
+public class PrismTransactionHandle
+        extends HiveTransactionHandle
+{
+    private final TransactionalMetadata transactionalMetadata;
```

Comment:
In general `ConnectorTransactionHandle` should be lightweight and JSON serializable. Including the entire `TransactionalMetadata` object as a handle seems not right.

Reply:
Yeah so I just made the TransactionalMetadata lie under a package-private method now (see presto-fb PR) and Jackson cant access it (We shouldnt be giving public access to this hack anyway). So it will serialize but it'll fail if we try to send transactionhandle over the wire for hive which we dont do.